interface/forms/procedure_order/common.php

```diff
@@ -427,6 +436,12 @@ function getListOptions($list_id, $fieldnames = array('option_id', 'title', 'seq
                         xlt("Order Successfully Sent") . "...\n" .
                         xlt("Order HL7 Content") .
                         ":\n" . $hl7 . "\n";
+                    if ($gbl_lab === 'quest') {
+                        $order_log .= xlt("Transmitting order to Quest");
+                        $ed->dispatch(new LabTransmitEvent($hl7), LabTransmitEvent::EVENT_LAB_TRANSMIT, 10);
```

Comment:
Where is LabTransmitEvent defined? I don't see it in this commit and I can't find it in the code system.

Reply:
@adunsulag It is in the preocedure_order/common.php line 441 and 442